inc/functions/files.php

```diff
@@ -818,15 +818,32 @@ function rocket_clean_domain( $lang = '' ) {
 	$urls = apply_filters( 'rocket_clean_domain_urls', $urls, $lang );
 	$urls = array_filter( $urls );
 
+	try {
+		$cache = new RecursiveDirectoryIterator( WP_ROCKET_CACHE_PATH, FilesystemIterator::SKIP_DOTS );
+	} catch ( UnexpectedValueException $e ) {
+		// No logging yet.
+		return;
+	}
+
+	try {
+		$iterator = new RecursiveIteratorIterator( $cache, RecursiveIteratorIterator::CHILD_FIRST );
+	} catch ( Exception $e ) {
+		// No logging yet.
+		return;
+	}
+
+	$dirs_to_preserve = get_rocket_i18n_to_preserve( $lang );
+	/** This filter is documented in inc/front/htaccess.php */
+	$url_no_dots = apply_filters( 'rocket_url_no_dots', false );
+
 	foreach ( $urls as $url ) {
 		$file = get_rocket_parse_url( $url );
```

Comment:
The code for `rocket_clean_domain()` is in PR #2510 . 

Until PR 2510 is merged, this PR is blocked.

Reply:
Merged here https://github.com/wp-media/wp-rocket/pull/2458/commits/71bb6dad18370618dd6f2fb57c9225f578fcccac